README.md

```diff
@@ -4,28 +4,38 @@
 
 ## Overview
 
-The SnowCLI is a command line interface for working with Snowflake. It allows users to create, manage, update, and view apps running in Snowflake. This is an open source project and contributions are welcome (though the project is maintained on a best-effort basis). We do plan to incorporate some of the patterns and features of this CLI into the Snowflake CLI (SnowSQL) in the future, and are hoping this project will help start the conversation on what a delightful developer experience could look like - and would love your help in shaping that with us.
+SnowCLI is a command line interface for working with Snowflake. It lets you create, manage, update, and view apps running in Snowflake.
+
+This is an open source project and contributions are welcome (though the project is maintained on a best-effort basis).
+
+We plan to incorporate some patterns and features of this CLI into the Snowflake CLI (SnowSQL) in the future. We hope this project starts a conversation about what a delightful developer experience could look like, and we'd love your help in shaping that with us!
+
+### Tour and quickstart
 
-### CLI tour and quickstart
 [![SnowCLI overview and quickstart demo](https://i.imgur.com/tqLVPWnm.png)](https://youtu.be/WDuBeAgbTt4)
 
-## Benefits
-- Supports local debugging and running of Snowflake apps.
+## Benefits of SnowCLI
+
+SnowCLI lets you locally run and debug Snowflake apps, and has the following benefits:
+
+- Has support for Snowpark Python **user defined functions** and **stored procedures**, **warehouses**, and **Streamlit** apps.
```

Comment:
I wasn't sure why this was a limitation, so I moved it to benefits. If there's a limitation here, I suggest rephrasing (is there something that people might expect to be supported that isn't?).

Reply:
Great point. Was initially a smaller list and calling out what wasn’t supported but now supports a lot :)